Tasks/Rusak/Task2/Task2-time.py

```diff
@@ -0,0 +1,79 @@
+import datetime as dt
+
+n2w_m = {
+1: ['одна', 'одной'], 2: ['две', 'двух'], 3: ['три', 'трех'],
+4: ['четыре', 'четырёх'], 5: ['пять', 'пяти'], 6: ['шесть', 'шести'],
+7: ['семь', 'семи'], 8: ['восемь', 'восьми'], 9: ['девять', 'девяти'],
+10: ['десять', 'десяти'], 11: ['одинадцать', 'одинадцати'], 
+12: ['двенадцать', 'двенадцати'], 13: ['тринадцать', 'тринадцати'],
+14: ['четырнадцать', 'четарнадцати'], 15: ['пятнадцать', 'пятнадцати'],
+16: ['шестнадцать'], 17: ['семнадцать'], 18: ['восемнадцать'], 19: ['девятнадцать'],
+20: ['двадцать'], 30: ['тридцать'], 40: ['сорок'], 50: ['пятьдесят']
+}
+
+n2w_h = {
+0: ['ноль'],
+1: ['час', 'первого'],
+2: ['два', 'второго'],
+3: ['три', 'третьего'],
+4: ['четыре', 'четвертого'],
+5: ['пять', 'пятого'],
+6: ['шесть', 'шестого'],
+7: ['семь', 'седьмого'],
+8: ['восемь', 'восьмого'],
+9: ['девять', 'девятого'],
+10: ['десять', 'десятого'],
+11: ['одинадцать', 'одинадцатого'],
+12: ['двенадцать', 'двенадцатого'],
+}
```

Comment:
good one! 

but I want to remind you about meaningful variables naming, n2w_m and n2w_h is not exactly clear